components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkConstants.java

```diff
@@ -106,6 +106,11 @@
     public static final String REMEMBER_ME_OPT_ON = "on";
     public static final String LAST_FAILED_AUTHENTICATOR = "LastFailedAuthenticator";
 
+    public static final String STATUS = "&status=";
+    public static final String STATUS_MSG = "&statusMsg=";
+    public static final String ACCOUNT_LOCKED_MSG = "ACCOUNT IS LOCKED";
+    public static final String ERROR_MSG = "This account is locked due to many failed login attempts.";
```

Comment:
Shall we change this to 'This account is locked due to exceeding maximum number of failed attempts'